backend/migrations/00072_add_resident_programs_feature.sql

```
@@ -0,0 +1,28 @@
+-- +goose Up
+-- +goose NO TRANSACTION
+-- A newly added enum value cannot be referenced inside the transaction that adds
+-- it, so this migration runs unwrapped and the INSERT below autocommits separately.
+ALTER TYPE feature ADD VALUE IF NOT EXISTS 'resident_programs';
+
+-- Resident-facing visibility of the Programs page is ON by default: once program
+-- tracking is enabled, residents keep access to their own program information
+-- unless an admin explicitly turns it off for their facility.
+INSERT INTO public.page_feature_flags (feature_flag_id, page_feature, enabled, created_at)
+SELECT id, 'resident_programs', TRUE, now()
+FROM public.feature_flags WHERE name = 'program_management'
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)